qase-pytest/src/qaseio/pytest/testops.py

```diff
@@ -45,7 +45,7 @@ def __init__(self,
         self.client = ApiClient(configuration)
         
         self.project_code = project_code
-        self.run_id = run_id
+        self.run_id = int(run_id)
```

Comment:
If we ignore the use of `--qase-to-run`, we will get an error of the following type:

`INTERNALERROR>   File "qase-python/qase-pytest/src/qaseio/pytest/testops.py", line 48, in __init__`
`INTERNALERROR>     self.run_id = int(run_id)`
`INTERNALERROR> ValueError: invalid literal for int() with base 10: ''`

Reply:
Fixed